### PR TITLE
practice link and button nav, display title in chart and footer

### PR DESCRIPTION
**Pull request by darrenmason**
image

### CODE REVIEW

File: src/components/learning-guide.cjsx [hunk 1]
```diff
@@ -1,21 +1,34 @@
...
+
   propTypes:
     courseId: React.PropTypes.any.isRequired
 
+  getInitialState: ->
+    unit: false
+
   navigateToPractice: (unit) ->
-    alert "TODO: Navigate to practice unit ID #{unit.id} (#{unit.title})"
+    console.log("Navigate to practice unit ID #{unit.id} (#{unit.title})")
```
Comment (philschatz) [inline src/components/learning-guide.cjsx]: can you just remove the console logs? Since they are no longer stubs?

Comment (darrenmason) [inline src/components/learning-guide.cjsx]: ok removed

File: src/components/learning-guide-chart.coffee [hunk 2]
```diff
@@ -48,10 +48,19 @@ module.exports = class LearningGuideChart
...
+    wrap = container.append('g')
+      .append("svg:text")
+      .attr("text-anchor","middle")
+      .attr("x", WIDTH/2)
+      .attr("y", 5)
+      .attr('class', 'main-title')
+      .text("Your Flight Path | "+guide.title+" | All Topics")
 
 
-  # Future improvement: Place clouds so they aren't
```
Comment (philschatz) [inline src/components/learning-guide-chart.coffee]: I was hoping for animated clouds

Comment (nathanstitt) [inline src/components/learning-guide-chart.coffee]: Hah! You know, I haven't mentioned it yet since I figured we had plenty else to work on, but another neat thing would be to animate the plane flying along the path between the circles.  It's quite doable: http://bl.ocks.org/KoGor/8162640

File: src/components/learning-guide.cjsx [hunk 3]
```diff
@@ -1,21 +1,32 @@
...
   propTypes:
     courseId: React.PropTypes.any.isRequired
 
+  getInitialState: ->
+    unit: false
+
   navigateToPractice: (unit) ->
-    alert "TODO: Navigate to practice unit ID #{unit.id} (#{unit.title})"
+    {courseId} = @props
+    @context.router.transitionTo('viewPractice', {courseId})
```
Comment (philschatz) [inline src/components/learning-guide.cjsx]: I think this may need to create a new practice in addition to `transitionTo` and send the tags (in the unit) to call it complete. But that can be a separate PR
